app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -34,25 +34,25 @@
-<%= presenter.attribute_to_html(:is_based_on_url, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:interactivity_type, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:learning_resource_type, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:typical_age_range, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:time_required, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:editor, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:has_volume) %>
-<%= presenter.attribute_to_html(:has_number) %>
-<%= presenter.attribute_to_html(:conference_location, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:conference_name, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:conference_section, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:has_journal, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:is_referenced_by, render_as: :faceted) %>
-<%= presenter.attribute_to_html(:isbn, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:contributor_advisor, render_as: :faceted, label: t('simple_form.labels.defaults.contributor_advisor')) %>
+<%= presenter.attribute_to_html(:contributor_committeemember, render_as: :faceted, label: t('simple_form.labels.defaults.contributor_committeemember')) %>
+<%= presenter.attribute_to_html(:degree_discipline, render_as: :faceted, label: t('simple_form.labels.defaults.degree_discipline')) %>
+<%= presenter.attribute_to_html(:degree_field, render_as: :faceted, label: t('simple_form.labels.defaults.degree_field')) %>
+<%= presenter.attribute_to_html(:degree_grantors, render_as: :faceted, label: t('simple_form.labels.defaults.degree_grantors')) %>
+<%= presenter.attribute_to_html(:degree_level, render_as: :faceted, label: t('simple_form.labels.defaults.degree_level')) %>
+<%= presenter.attribute_to_html(:degree_name, render_as: :faceted, label: t('simple_form.labels.defaults.degree_name')) %>
+<%= presenter.attribute_to_html(:graduation_year, render_as: :faceted, label: t('simple_form.labels.defaults.graduation_year')) %>
+<%= presenter.attribute_to_html(:is_based_on_url, render_as: :faceted, label: t('simple_form.labels.defaults.is_based_on_url')) %>
+<%= presenter.attribute_to_html(:interactivity_type, render_as: :faceted, label: t('simple_form.labels.defaults.interactivity_type')) %>
+<%= presenter.attribute_to_html(:learning_resource_type, render_as: :faceted, label: t('simple_form.labels.defaults.learning_resource_type')) %>
+<%= presenter.attribute_to_html(:typical_age_range, render_as: :faceted, label: t('simple_form.labels.defaults.typical_age_range')) %>
+<%= presenter.attribute_to_html(:time_required, render_as: :faceted, label: t('simple_form.labels.defaults.time_required')) %>
+<%= presenter.attribute_to_html(:editor, render_as: :faceted, label: t('simple_form.labels.defaults.editor')) %>
+<%= presenter.attribute_to_html(:has_volume, render_as: :faceted, label: t('simple_form.labels.defaults.has_volume')) %>
```

Comment:
Since has_volume is not searchable, I think `presenter.attribute_to_html(:has_volume, label: t('simple_form.labels.defaults.has_volume'))` would be better here, so that we don't get a link to search.